### ISSUE TITLE
App still flickers?

### ISSUE THREAD

**Issue by lfDev28** [DiMatteoL/buzzrank-tutorial#1]
Any idea why my app might still be flickering? Believe I followed everything right

**Comment by DiMatteoL** [comment]
Hey, Next went through a major version change (Next 13), so it might be the reason ?
I'd need more information to help you further.

**Comment by lfDev28** [comment]
Yeah I’m on v12 still tho. Got a discord? Just thought I’d see if you could help lol

**Comment by lfDev28** [comment]
> Hey, Next went through a major version change (Next 13), so it might be the reason ?
> I'd need more information to help you further.

yeah.  I’m on v12 still tho. Got a discord? Just thought I’d see if you could help lol

**Comment by alexrabin** [comment]
Still flickering for me too
